meteor-app/imports/ui/config/bike-assessment-schemas.js

```diff
@@ -135,7 +136,9 @@ export default [{
         },
         pickUpDate: {
           type: "string",
-          title: "Pick-up Date"
+          title: "Pick-up Date",
+          format: "date",
+          default: "2018-10-10"
```

Comment:
Everything is going to be ready on the 10th of October this year? It should also be a date, not a string.

Reply:
Maybe you haven't read through the docs for json schema form. That is the date format used to specify the default date. Also, the date here is just a placeholder.

Reply:
Fair enough I suppose, but it looks sus to me